platforms/hyperledger-indy/scripts/indy-ledger-script/indy-ledger.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+vault_addr=$1
+vault_token=$2
+admin_path=$3
+admin_name=$4
+identity_path=$5
+identity_name=$6
+identity_role=$7
+pool_genesis_path=$8
+
+[ -z "$vault_addr" ] && { echo "Script Failed: Vault Address Missing"; exit 1;} || echo "Vault Address Received";
+[ -z "$vault_token" ] && { echo "Script Failed: Vault Token Missing"; exit 1;} || echo "VAult Token Received";
+[ -z "$admin_path" ] && { echo "Script Failed: Admin DID Path Missing"; exit 1;} || echo "Admin DID Path Received";
+[ -z "$admin_name" ] && { echo "Script Failed: Admin Name Missing"; exit 1;} || echo "Admin Name Received";
+[ -z "$identity_path" ] && { echo "Script Failed: Identity DID Path Missing"; exit 1;} || echo "Identity DID Received";
+[ -z "$identity_name" ] && { echo "Script Failed: Identity DID Name Missing"; exit 1;} || echo "Identity Name Received";
+[ -z "$identity_role" ] && { echo "Script Failed: Identity DID Name Missing"; exit 1;} || echo "Identity Role Received";
+[ -z "$pool_genesis_path" ] && { echo "Script Failed: Genesis Pool File Path Missing"; exit 1;} || echo "Genesis Pool File Path Received";
+
+export VAULT_ADDR=$vault_addr
+export VAULT_TOKEN=$vault_token
+
+admin_did=$(vault kv get -field=did $admin_path/$admin_name)
```

Comment:
Do you even have the vault CLI in this Docker image? I don't see it installed in the Dockerfile

Reply:
It is assumed that vault is already installed on the system on which this script is running
Thats why I haven't installed

Reply:
The script runs in the indy-cli Docker image